backend/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app');
+var debug = require('debug')('backend:server');
+var http = require('http');
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+var port = normalizePort(process.env.PORT || '3000');
```

Comment:
we should prob change 3000 to 9000 but we can also do that in a separate merge

Reply:
Ohh sounds good. What is the best practice to for handling project_module? I think I should just remove both project_module and the package_lock file because they can be generated when you npm install. And when we push, we don’t push them just the package. Sure, I will change the local host to 9000. Then if no problem, we can merge it :)

Reply:
great! yeah maybe we can just get rid of the node_modules and then we can just npm install in their respective folders 👍 maybe just make sure it's working first ahaha

Reply:
ok May, can you check if it works now. If it does, please merge it

Reply:
thx :)

Reply:
could you please fix the merge conflicts😅